### DISCUSSION TITLE
Ability to edit a post

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1396] [Protocol (atproto)]
It would be nice if that was added soon so that way, I don't have to delete and then re-post it. Which will save time.

**Comment by dholms** [comment]
Hey there, this discussion board is meant to support high-level discussion of the protocol or specific features.

Many of your posts here are just one sentence feature requests which just clouds up the forum. While the subject of this post is appropriate to discuss (editing posts), I would encourage you to flesh out thoughts & set a groundwork for discussion rather than just a short feature request.

**Reply by Zero3K** [reply]
How do I do that?

**Reply by Zero3K** [reply]
Where do I post it as an Issue?

**Reply by venteto** [reply]
There is an [open issue here](https://github.com/bluesky-social/social-app/issues/673) if you want to follow it
